### PR TITLE
Arxivce 805 html route

### PR DESCRIPTION
**Pull request by mnazzaro**
This branch includes the latest HTML conversion styles as well as the capability to serve classic html papers and conversions from browse in GCP. It also updates the sync script to handle html submissions and has some minor changes to the abstract page.

### CODE REVIEW

File: browse/templates/abs/extra_services.html [hunk 1]
```diff
@@ -52,20 +52,16 @@
...
       {%- endif -%}
       {%- if format == 'latexml' -%}
-  <li>
-    <a href="{{ latexml_url }}" class="abs-button" id="latexml-download-link">HTML (experimental)</a>
-  </li>
+  <li><a href="{{ latexml_url }}" class="abs-button" id="latexml-download-link">HTML (experimental)</a></li>
       {%- endif -%}
       {%- if format == 'src' and current-%}
-  <li><a href="{{url_for('src.e_print', arxiv_id=requested_id)}}" class="abs-button download-eprint">TeX Source</a></li>
+  <li><a href="{{url_for('src.src', arxiv_id=requested_id)}}" class="abs-button download-eprint">TeX Source</a></li>
```
Comment (bdc34) [inline browse/templates/abs/extra_services.html:58]: Do not change to src unless there is a Jira ticket that explicitly requests this.

File: browse/templates/abs/extra_services.html [hunk 2]
```diff
@@ -52,20 +52,16 @@
...
       {%- elif format.startswith('ps') -%}
   <li><a href="{{url_for('dissemination.ps', arxiv_id=requested_id)}}" class="abs-button download-ps">PostScript{% if format.endswith('(400)') %} (400 dpi){% elif format.endswith('(600)') %} (600 dpi){% elif format.endswith('(cm)') %} (Type I cm){% elif format.endswith('(CM)') %} (Type I CM){% endif %}</a></li>
       {%- elif format == 'dvi' -%}
   <li><a href="{{url_for('dissemination.dvi', arxiv_id=requested_id)}}" class="abs-button download-dvi">{{ format.upper() }}</a></li>
       {%- elif format == 'html' -%}
   <li><a href="{{url_for('dissemination.html', arxiv_id=requested_id)}}" accesskey="f" class="abs-button download-html">{{ format.upper() }}</a></li>
-      {%- elif format == 'other' -%}
```
Comment (bdc34) [inline browse/templates/abs/extra_services.html:67]: This removal is not in master or develop. I'm not sure why adding the HTML should remove the "Other Formats" button.

File: browse/templates/abs/extra_services.html [hunk 3]
```diff
@@ -189,11 +185,14 @@ <h2>Access Paper:</h2>
       <div class="abs-license">
         {%- if withdrawn -%}
         <div hidden>No license for this version due to withdrawn</div>
-        {%- elif abs_meta.license.icon_uri_path -%}
-        <a href="{{abs_meta.license.effective_uri}}" title="Rights to this article"><img src="https://arxiv.org{{ abs_meta.license.icon_uri_path }}"/></a>
         {%- else -%}
-        (<a href="{{abs_meta.license.effective_uri}}" title="Rights to this article">view license</a>)
+          {%- if 'other' in formats -%}
```
Comment (bdc34) [inline browse/templates/abs/extra_services.html:189]: This code is not in master or develop and doesn't seem to have anything to do with HTML-papers.

**Review by bdc34 — CHANGES_REQUESTED** [review]
There were some changes unrelated to html-papers in the extra_services.html. If these are not part of the HTML papers work, please don't include them.

**Comment by bdc34** [comment]
Looks good.

**Review by bdc34 — APPROVED** [review]

File: browse/static/css/latexml_styles.css [hunk 4]
```diff
@@ -1058,33 +971,198 @@ article{
...
+    .ltx_page_main > .ltx_page_content{
+      margin: 6.5em 2em 3.5em 2em;
+      padding: 1em;
+    }
+    .ltx_page_main >.ltx_TOC.active {
+      max-width: 15%;
+    }
+    .ltx_page_main > .ltx_TOC:not(.active) {
+      margin-right: -30px;
+    }
```
Comment (SBBCornell) [inline browse/static/css/latexml_styles.css:1162]:
```suggestion
    }
    .ltx_page_main > .ltx_TOC {
        padding-top: 10vh;
    }
```

**Comment by bdc34** [comment]
I suspect this PR can be closed due to this code being merged from other places. These features are live on the site.

**Comment by bdc34** [comment]
Mark says these are already live and this can be closed.

**Pull request closed without merging**